dsa-problems/gfg-problems/basic/0201-0300.md

```diff
@@ -0,0 +1,623 @@
+    "problemName": "The Spider Steps"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/minimize-sum-of-alternate-product2033/1",
+    "solutionLink": "#",
+    "problemName": "Minimize sum of alternate product"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/factors-sum2016/1",
+    "solutionLink": "#",
+    "problemName": "Factors Sum"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/finger-game1755/1",
+    "solutionLink": "#",
+    "problemName": "Finger Game"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/sum-of-fifth-powers-of-the-first-n-natural-numbers3415/1",
+    "solutionLink": "#",
+    "problemName": "Sum of fifth powers of the first n natural numbers"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/c-classes-introduction/1",
+    "solutionLink": "#",
+    "problemName": "C++ Classes Introduction"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/fake-profile3906/1",
+    "solutionLink": "#",
+    "problemName": "Fake Profile"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/xor-of-all-elements0736/1",
+    "solutionLink": "#",
+    "problemName": "XOR of all elements"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/product-sum3012/1",
+    "solutionLink": "#",
+    "problemName": "Product Sum"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/magical-string3653/1",
+    "solutionLink": "#",
+    "problemName": "Magical String[Duplicate Problem]"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/java-collection-set-3-hashmap/1",
+    "solutionLink": "#",
+    "problemName": "HashMap Operations"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/student-record1752/1",
+    "solutionLink": "#",
+    "problemName": "Student record"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/making-elements-distinct3139/1",
+    "solutionLink": "#",
+    "problemName": "Making elements distinct"
+  },
+  {
+    "difficulty": "Basic",
+    "leetCodeLink": "https://www.geeksforgeeks.org/problems/maximum-integer-value1434/1",
+    "solutionLink": "#",
+    "problemName": "Maximum Integer Value "
+  }
+]
+
+
+<Table 
+    title=""
+    data={problems}
+    isSorted={false}
+    collectionLink="https://www.geeksforgeeks.org/"
+/>
+
+Now, you can see the list of problems in a table format. You can click on the problem link to view the problem on the LeetCode website. You can also click on the solution link to view the solution of the problem.
```

Comment:
Change name `LeetCode website` to `GeeksForGeeks website`

Reply:
ok